nowcasting_dataset/data_sources/gsp/eso.py

```diff
@@ -21,10 +21,19 @@
 import geopandas as gpd
 import pandas as pd
 
-from nowcasting_dataset.geospatial import WGS84_CRS
+from nowcasting_dataset.geospatial import osgb_to_lat_lon, WGS84_CRS
+
 
 logger = logging.getLogger(__name__)
 
+rename_save_columns = {
```

Comment:
Maybe in another PR, we could make it so we don't have to rename the columns, but instead we use the same column names throughout the code?

Reply:
Yea I did this as the column names seemed to be limited to 10, but let me check this again

Reply:
https://en.wikipedia.org/wiki/Shapefile#Limitations and https://github.com/geopandas/geopandas/issues/1417
says its limited to 10

Reply:
And then its a balance of a bit of extra code here to give full verbose, compared to a shorter column name and it not being as verbose. Personally I like the fully verbose version

Reply:
Ah, sorry, my mistake... I hadn't realised shapefiles have these limitations!  Thanks for explaining.  Cool, let's leave the code as-is!